resources/main-template.cc

```diff
@@ -311,7 +313,7 @@ static int RunNodeInstance(MultiIsolatePlatform* platform,
     if (LoadBoxednodeEnvironment(context).IsEmpty()) {
       return 1; // There has been a JS exception.
     }
-    boxednode::MarkTime("Loaded Environment, entering loop");
+    boxednode::MarkTime("Node.js Instance", "Loaded Environment, entering loop");
```

Comment:
I'm not sure if this + the one above is really makes sense to group in with the Node.js Instance bootstrapping, it's really our own code here (`LoadBoxednodeEnvironment` runs the synchronous part of the main script + microtask queue, everything before the event loop takes over)

Reply:
Maybe I can change the name to Node.js Configuration or something like that.

Reply:
Changed to "Boxednode Binding", which with your explanation, sounds more accurate.